admin/src/app/search/search.component.ts

```diff
@@ -33,6 +33,7 @@ import { StateService } from '@admin-core/services/state.service';
     styleUrls: ['./search.component.scss']
 })
 export class SearchComponent implements OnInit, OnDestroy {
+    readonly projectPlanCodeEnum = ProjectPlanCodeEnum;
```

Comment:
also the formatting on this line, the indent

Reply:
My IDE default to 'tab' 4 spaces... sometimes I forgot to change it on IDE. Should be good now.

Reply:
Should be good now.